package.json

```diff
@@ -24,7 +24,7 @@
   "peerDependencies": {
     "code-push": ">=1.8.0-beta",
     "cordova-plugin-file-transfer": ">=1.3.0",
-    "cordova-plugin-file": ">=3.0.0",
+    "cordova-plugin-file": "^3.0.0",
```

Comment:
I would vote for leaving this for now. As Scott mentioned, this fix is necessary for other reasons, so I'd rather change this dependency in another PR if we needed to.